lxd/storage_volumes_backup.go

```diff
@@ -336,27 +333,25 @@ func storagePoolVolumeTypeCustomBackupsPost(d *Daemon, r *http.Request) response
 		return resp
 	}
 
-	var poolID int64
-
-	err = s.DB.Cluster.Transaction(r.Context(), func(ctx context.Context, tx *db.ClusterTx) error {
-		var err error
-
-		poolID, _, _, err = tx.GetStoragePool(ctx, poolName)
-
-		return err
-	})
+	pool, err := request.GetCtxValue[storagePools.Pool](r.Context(), ctxStoragePool)
 	if err != nil {
 		return response.SmartError(err)
 	}
 
-	resp = forwardedResponseIfVolumeIsRemote(s, r, poolName, projectName, volumeName, cluster.StoragePoolVolumeTypeCustom)
-	if resp != nil {
-		return resp
+	// FIXME: Forward if target parameter is remote?
```

Comment:
Again here there doesn't seem to be any forwarding if the target is not this cluster member, but there is forwarding if the volume is located on another member.

Reply:
I've added an issue to track this here: https://github.com/canonical/lxd/issues/13548